client/jetpack-connect/plans.jsx

```diff
@@ -188,6 +189,12 @@ class Plans extends Component {
 		);
 	}
 
+	handleInfoButtonClick = info => {
+		this.props.recordTracksEvent( 'calypso_jpc_help_' + info + '_click', {
```

Comment:
Programmatically creating event names is not recommended - they should be hardcoded so they are easily searchable in the code base.

Reply:
ofc! good point.